.p4-env.json

```diff
@@ -0,0 +1,16 @@
+{
+  "planet4": {
+    "content": {
+      "db": "v0.2.18",
+      "images": "1-25"
+    },
+    "repos": {
+      "planet4-base": "main",
+      "planet4-master-theme": "main",
+      "planet4-plugin-gutenberg-blocks": "fix/columns-tests-timing"
```

Comment:
Should we reset the branch here?

Reply:
right, just waiting for [that pr](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1062) to be merged to give this test a better chance to pass :grimacing: